packages/world-modules/package.json

```diff
@@ -10,8 +10,15 @@
   "license": "MIT",
   "type": "module",
   "exports": {
-    "./out/*": "./out/*",
-    "./*": "./dist/*"
+    "./internal/mud.config": "./dist/mud.config.js",
+    "./out/*": "./out/*"
+  },
+  "typesVersions": {
+    "*": {
+      "internal/mud.config": [
+        "./mud.config.ts"
```

Comment:
I am surprised I needed this, but I guess its because the `.ts` file is at the same path as the import for world/store packages.